src/modules/todo/todo.controller.ts

```diff
@@ -20,7 +21,15 @@ export class TodoController {
   public async userTodos(req: Request) {
     //@ts-ignore
     const { _id } = req.user;
-    return await this.todoService.filterTodo({ userId: _id });
+    let userTodos = await redisCheckName('userTodos');
+
+    if (!!userTodos) {
+      return JSON.parse(userTodos);
+    } else {
+      const result = await this.todoService.filterTodo({ userId: _id });
+      await redisSet('userTodos', JSON.stringify(result));
```

Comment:
Will it work for multiple users?

Reply:
I logged in with another user profile and checked - it's work =)

Reply:
@joseph-dev  Yep you right it's not work correct for multiple users
this should fix this
<img width="634" alt="Screenshot 2024-01-14 at 16 39 19" src="https://github.com/ArtyomZayarny/todo-api/assets/15728688/5c061d5e-6043-460b-86db-53f43abd00f1">